manifests/key.pp

```diff
@@ -56,9 +56,15 @@
     mode   => '0750',
   }
 
+  if ($facts['os']['distro']['codename'] == 'jammy') {
```

Comment:
Instead of this, you could just define a path for the exec below,

```
exec {"...":
   ...
   path => ['/usr/bin', '/usr/sbin'],
}
```

and then use just opendkim-genkey without the absolute path in the command

Reply:
Applied your suggestion